src/Keboola/DbExtractor/TableResultFormat/TableColumn.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor\TableResultFormat;
+
+use Keboola\DbExtractor\Exception\UserException;
+use Keboola\Utils;
+
+class TableColumn
+{
+    private string $name;
+
+    private string $sanitizedName;
+
+    private string $type;
+
+    private bool $primaryKey = false;
+
+    private bool $uniqueKey = false;
+
+    private string $length;
+
+    private bool $nullable;
+
+    private string $default;
+
+    private int $ordinalPosition;
+
+    private string $description;
+
+    private bool $autoIncrement;
```

Comment:
myslim se ze tohle uz nepouzivame?

Reply:
používá se např:
https://github.com/keboola/db-extractor-mssql/blob/master/src/Keboola/DbExtractor/Extractor/MetadataProvider.php#L264

ale i v jiných extractorech

Reply:
jo, but what I meant was that I initial used this as part of incremental fetching logic but it is not used that way anymore.  Maybe it is still interesting metadata though so we can keep it.